docs/intellij-quickstart.md

```diff
@@ -1,7 +1,7 @@
 # IntelliJ quickstart
 
 ## JVM
-OpenJDK12 is strongly recommended. You can [download it from IntelliJ](https://www.jetbrains.com/help/idea/sdk.html)
+OpenJDK 17 is strongly recommended. You can [download it from IntelliJ](https://www.jetbrains.com/help/idea/sdk.html)
```

Comment:
OpenJDK 17 is now mandatory as of mainnet release `v0.26.x`. I would suggest changing this to say `required` or similar verbiage.

Reply:
Additionally, I would suggest updating the SDK download URL to link to the Adoptium builds.

Reply:
> OpenJDK 17 is now mandatory as of mainnet release `v0.26.x`. I would suggest changing this to say `required` or similar verbiage.

Ok, we should update the root [README](https://github.com/hashgraph/hedera-services/blob/master/README.md#jvm) also